### PR TITLE
Add a timeout to the VCR proxy call

### PR DESCRIPTION
**Pull request by blampe**
@abhinav (shrug)

### CODE REVIEW

**Comment by abhinav** [comment]
:+1:

File: tchannel/testing/vcr/patch.py [hunk 1]
```diff
@@ -103,6 +103,7 @@ def send(self, arg1, arg2, arg3,
             vcr_response_future = tchannel.thrift(
                 proxy.VCRProxy.send(vcr_request),
                 hostport=self.vcr_hostport,
+                timeout=1.0,
```
Comment (breerly) [inline tchannel/testing/vcr/patch.py]: - you'll want this to be configurable by those using vcr
- seems like a low default

Comment (abhinav) [inline tchannel/testing/vcr/patch.py]: Oh, you're right. If a recording does not exist, a real request will be made so we should use the timeout from the original request + some delay for the proxy request.

Comment (abhinav) [inline tchannel/testing/vcr/patch.py]: Created https://github.com/uber/tchannel-python/pull/403

**Pull request merged**